### PR TITLE
[COREV] Add CI workflow file corev-llvm-tests.yml.

### PR DESCRIPTION
**Pull request by xingmingjie**

### CODE REVIEW

**Comment by xingmingjie** [comment]
The PR should be updated later.

File: .github/workflows/corev-llvm-tests.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+
+jobs:
+  check_all:
+    # if: github.repository_owner == 'llvm'
```
Comment (jeremybennett) [inline .github/workflows/corev-llvm-tests.yml]: Uncomment this and restrict only to organization `openhwgroup`

**Review by jeremybennett — CHANGES_REQUESTED** [review]

**Comment by jeremybennett** [comment]
Suggest we should disable Windows for the time being, so we can merge this.

We should then raise an issue to cover adding back Windows support in the future.

**Comment by simonpcook** [comment]
These jobs all take a long time because caches are cold, is it possible to [increase timeouts](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) instead?

**Comment by xingmingjie** [comment]
> These jobs all take a long time because caches are cold, is it possible to [increase timeouts](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) instead?

See https://github.com/github/docs/issues/7984. The jobs.<job_id>.timeout-minutes description is misleading, we can't set a value larger than 360 when using GitHub hosted runners. So I just disabled windows for the time being.

File: .github/workflows/corev-llvm-tests.yml (hunk 1 above)
Comment (jeremybennett) [reply .github/workflows/corev-llvm-tests.yml]: Thanks for updating this.

**Review by jeremybennett — APPROVED** [review]
This is now good to merge.

**Comment by jeremybennett** [comment]
@xingmingjie I am going to merge this, even though the tests failed, because the failure was in one of the tests, not in the CI infrastructure itself.

**Pull request merged**